deploy/cephcsi/image/Dockerfile

```diff
@@ -1,22 +1,27 @@
 ARG SRC_DIR="/go/src/github.com/ceph/ceph-csi/"
 ARG GO_ARCH
-ARG BASE_IMAGE=ceph/ceph:v15
+ARG BASE_IMAGE
 
 FROM ${BASE_IMAGE} as builder
 
 LABEL stage="build"
 
-ARG GOLANG_VERSION=1.13.9
 ARG CSI_IMAGE_NAME=quay.io/cephcsi/cephcsi
 ARG CSI_IMAGE_VERSION=canary
 ARG GO_ARCH
 ARG SRC_DIR
 ARG GIT_COMMIT
 ARG GOROOT=/usr/local/go
 
-RUN mkdir -p ${GOROOT} && \
+COPY build.env /
+
+RUN source /build.env && \
+    mkdir -p ${GOROOT} && \
     curl https://storage.googleapis.com/golang/go${GOLANG_VERSION}.linux-${GO_ARCH}.tar.gz | tar xzf - -C ${GOROOT} --strip-components=1
 
+# test if the downloaded version of Golang works (different arch?)
```

Comment:
is it required?

Reply:
It is just an additional check. While working on this PR, I hit an issue where the amd64 version of Golang was installed in the arm64 image. Errors were pretty unclear, this check causes it to abort early.